xml/System.Diagnostics/Activity.xml

```diff
@@ -40,8 +40,8 @@
         <Parameter Name="operationName" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="operationName">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="operationName">The name of the operation.</param>
+        <summary>Note that <see cref="T:System.Diagnostics.Activity" /> has a 'builder' pattern, where you call the constructor, a number of 'Set*' and 'Add*' APIs and then call <see cref="M:System.Diagnostics.Activity.Start" /> to build the activity.  You MUST call <see cref="M:System.Diagnostics.Activity.Start" /> before using it.</summary>
```

Comment:
I would add a comma here (after `APIs`) for clarity:

> has a 'builder' pattern, where you call the constructor, a number of 'Set*' and 'Add*' APIs, and then call...

Reply:
```suggestion
        <summary>Note that <see cref="T:System.Diagnostics.Activity" /> has a 'builder' pattern, where you call the constructor, a number of 'Set*' and 'Add*' APIs, and then call <see cref="M:System.Diagnostics.Activity.Start" /> to build the activity.  You MUST call <see cref="M:System.Diagnostics.Activity.Start" /> before using it.</summary>
```